esp/services/ldapenvironment/ldapenvironmentService.cpp

```diff
@@ -155,7 +150,7 @@ bool CldapenvironmentEx::createSecret(SecretType type, const char * secretName,
             break;
     }
 
-    VStringBuffer cmdLine("%s\"%s\"", cmdLineSafe.str(), pwd);
+    VStringBuffer cmdLine("%s%s", cmdLineSafe.str(), pwd);
```

Comment:
Is this reversing your prior fix for extended characters in the password?

Reply:
Yes it is reversing previous fix.  As it turns out, the double quotes became part of the password. I was not aware of this because I did a copy/paste of the generated helm. But when I  had to manually type in the password if became painfully obvious. Also obvious when dumping the password with
kubectl get secret adminLDAP-INSURANCE -o jsonpath="{.data.password}" | base64 --decode

Reply:
Ok, does the other JIRA need to be reopened to provide a different fix?

Reply:
@afishbeck I would think not, since this new service is not in use yet